### ISSUE TITLE
Add Ozone (original version)

### ISSUE THREAD

**Issue by MpegEVIL** [OvercastNetwork/Rotations#449]

**Comment by Ipskie** [comment]
please provide rationale for this request

**Comment by MpegEVIL** [comment]
I want to play it.

**Comment by CoWinkKeyDinkInc** [comment]
That's not a rational reason.

**Comment by MpegEVIL** [comment]
There are no maps in the rotation that have the ender pearl/bow feature, which was a lot of fun.

**Comment by DoomRobotBoy** [comment]
@MpegEVIL I think the map was removed due to low player counts on it. Saying you liked it can't bring it back because there is no guarantee others will follow your example. 

I liked it myself as well, but not with the low player counts. TDM seems to be ignored mostly these days so I suggest spicing Ozone up with control points.

**Comment by MpegEVIL** [comment]
I don't remember low player counts on the map at all. Whenever it turned up in the rotations, it filled up almost instantly. It was often difficult to get a slot.

**Comment by DoomRobotBoy** [comment]
When did you supposedly play these filled up matches? Because I always switched to Ozone whenever it was up and had over 25 players.

**Comment by MpegEVIL** [comment]
I believe it was a couple years ago. I took a hiatus from Overcast so maybe people lost interest.

At the very least, I'd suggest implementing the enderpearl/bow feature onto another map.

**Comment by DoomRobotBoy** [comment]
The problem is that a map needs to be of a medium to large size for that. And we don't generally have such maps anymore (as in large maps). Reason for this is that player counts have been shrinking quite aggressively since 1 year ago. Ozone is currently the only map that can have the feature and will probably be the only map that will have it for at least a few months. Best to do is either motivate players to get interested in it again, which can imo only be done by spicing the map itself up.

**Comment by MpegEVIL** [comment]
Any chance you could send me the map and any plugins required to run it? Was it entirely command-block based?

**Comment by DoomRobotBoy** [comment]
Nope. Its based on plugins and I don't think there is a public download.

**Comment by Electroid** [comment]
Maybe we'll revive it as FFA.

**Comment by MpegEVIL** [comment]
That would be cool!

**Comment by tanpara** [comment]
I think that a free-for-all version would certainly be better than the usual one, it'd have a lot more use of space. That's a given.

But I may recommend sizing down the map a bit, it's still a bit large even for what would be a FFA. I'd consider using the system Nadastorm used and scale it down slightly, while still retaining the same gameplay.

**Comment by MpegEVIL** [comment]
I feel like it would work great as a free-for-all at it's current size. If
it were made smaller, it would be a lot more claustrophobic. One would
struggle to get a good pick in without dying instantly.

**Comment by izsaf** [comment]
Use of the bow makes the large map size reasonable, though it does make little sense for a TDM. If it was changed to FFA with random spawn points (maybe remove the spawn towers?) it would be fine since the map seemed to work well enough if a bit sparse in TDM. Simply switching the mode would effectively double the encounters.

Most importantly, if the map was made smaller you wouldn't have as much room to teleport around and find good vantage points and such.

Ozone was my personal favorite map, (I think I first found Overcast because of a video of SethBling playing it), and it would be great to see the different DM style that it had to any other map.
